Model/Checkout/PlaceOrder.php

```diff
@@ -200,6 +201,9 @@ public function execute(
             $this->quoteRepository->save($quote);
         }
 
+        $quote->setPaymentMethod(ConfigProvider::CODE);
+        $quote->getPayment()->importData(['method' => ConfigProvider::CODE]);
+
```

Comment:
Is it possible to move this to line 146? I am not sure which line gives the crash.

Reply:
Done. I believe the underlying error is only thrown once the quote is submitted. But this way we're always safe I suppose